README.md

```diff
@@ -68,7 +64,7 @@ After you're finished please remove all the comments and instructions!
 
 # 📖 setup and mobile first Project <a name="about-project"></a>
 
-setup and mobile first is a simple personal portfolio project using mobile first approach to make it look good on mobile/
+This is my personal portfolio project with mobile first approach. I will be add animation and more feature later on.
```

Comment:
- [ ] Please use a descriptive title for this header to describe your project not only the current milestone for professional documentation e.g. 👇🏽 

| Correct ✔️ | Incorrect ❌ |
|---|---|
|📖 Portfolio-project |  📖  Portfolio: desktop version: day 2 |
**Portfolio project** is the first app that will contain my bests projects... | In this part of the project, I will do the desktop version of my portfolio to continue adding more features. |